src/upcoming-components/Accordion/Accordion.tsx

```diff
@@ -1,35 +0,0 @@
-import clsx from 'clsx';
-import React, { ReactNode } from 'react';
-import styles from './Accordion.module.css';
-import AccordionPanel from '../AccordionPanel';
-
-export interface Props {
-  /**
-   * Child node(s) that can be nested inside component
-   */
-  children?: ReactNode;
-  /**
-   * CSS class names that can be appended to the component.
-   */
-  className?: string;
-}
-
-/**
- * BETA: This component is still a work in progress and is subject to change.
- *
- * ```ts
- * import {Accordion} from "@chanzuckerberg/eds";
- * ```
- *
- * TODO: update this comment with a description of the component.
- */
-export const Accordion = ({ children, className, ...other }: Props) => {
-  const componentClassName = clsx(styles['accordion'], className);
-  return (
-    <dl className={componentClassName} {...other}>
-      {children}
-    </dl>
```

Comment:
I think this is a good reason to remove the accordion component.

This suggests that the accordion uses will be descriptive in nature when that may not be the case in use cases and can be applying the wrong semantic element. 

Even the [w3 accordion](https://www.w3.org/WAI/ARIA/apg/example-index/accordion/accordion) example shows form elements in the panel

Reply:
Good callout. This is also a nice way to document concerns about any of these as we go. Anyone referring to this PR will see the discussion and can make informed choices.